run.sh

```diff
@@ -9,48 +9,48 @@ set -u
 # Make sure to kill all background tasks when exiting.
 trap "kill 0" EXIT
 
-TransportSecurityProtocols=( noise plaintext )
+TcpTransportSecurityProtocols=( noise plaintext )
 
 echo "# Start Rust and Golang servers."
 ./rust/target/release/server --private-key-pkcs8 rust/test.pk8 --listen-address /ip4/0.0.0.0/tcp/9992 > /dev/null 2>&1 &
```

Comment:
I am not quite sure whether it worked for QUIC at all but the addresses should be in the form of `/ipX/Y/udp/Z/quic`.